devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy

```diff
@@ -14,12 +14,15 @@
 metadata {
 	definition (name: "Z-Wave Switch Generic", namespace: "smartthings", author: "SmartThings") {
 		capability "Actuator"
+		capability "Health Check"
  		capability "Switch"
 		capability "Polling"
 		capability "Refresh"
 		capability "Sensor"
 
 		fingerprint inClusters: "0x25", deviceJoinName: "Z-Wave Switch"
+		fingerprint mfr:"001D", prod:"1A02", model:"0334", deviceJoinName: "Leviton DZPA1 Plug-in Appliance Module"
```

Comment:
I've modified the deviceJoinName to "GE Plug-in Outdoor Switch" for the GE switch. For the Leviton DZPA1 Plug-in Appliance Module which we have, I've changed the deviceJoinName to "Leviton Appliance Module" with fingerprint mfr:"001D", prod:"1A02", model:"0334".

@mckeed Do you want to keep this DZPA1 fingerprint in this PR or should I remove this fingerprint as well from this PR? Please confirm.